app/views/docs/command-line.phtml

```diff
@@ -138,15 +138,15 @@ brew install --HEAD appwrite</code></pre>
     "collections": [
         {
             "$id": "6213bb2d567d8a206614",
-            "$read": [
-                "team:admin"
-            ],
-            "$write": [
-                "team:admin"
+            "$permissions": [
+                "read(any)",
+                "create(team:core)",
+                "update(team:core)",
+                "delete(team:core)",
```

Comment:
I think it is `read("any")`, not `read(any)`. That would follow query syntax.

Reply:
Let's wait until I can try this :') I have no idea